backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -2585,8 +2589,17 @@ SDL_Surface *SurfaceSdlGraphicsManager::SDL_SetVideoMode(int width, int height,
 
 	_renderer = SDL_CreateRenderer(_window->getSDLWindow(), -1, rendererFlags);
 	if (!_renderer) {
-		deinitializeRenderer();
-		return nullptr;
+		if (_vsync) {
```

Comment:
We could check for rendererFlags & SDL_RENDERER_PRESENTVSYNC here, no need for a separate flag